DescompoNumero.py

```diff
@@ -0,0 +1,37 @@
+#definimos la funcion que traiga los primos que hay hasta cierto numero
+ListaPrimos = []
+def Primos(HastaNumero):
+    for i in range(2, HastaNumero+1):
+        for j in range(2, i):
+            if (i % j == 0):
+                break
+        else:
+            ListaPrimos.append(i)
```

Comment:
It's not indented correctly, try to use a linter for python such as `black `or `flake8 `